api/app/signals/apps/reporting/models/tdo.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.gis.db import models
+
+
+class TDOSignal(models.Model):
```

Comment:
Maybe add a comment explaining that this model belongs to a database view? Not a blocker